helm/endpoint-operator-chart/Chart.yaml

```diff
@@ -0,0 +1,2 @@
+name: cluster-operator-chart
+version: 0.0.1-[[ .SHA ]]
```

Comment:
The Helm magic could be separated as well.

Reply:
I also think this PR is too big. I'd separate helm, service stuff. I'd add minimal set of dependencies (no kubernetes deps). vendor dir will change anyway when you add importing code.